tests/fixtures/sdk_code/feature_with_cleaned_tz_offset_column.py

```diff
@@ -20,8 +20,9 @@
 )
 col = event_view["event_timestamp"]
 col_1 = event_view["tz_offset"]
-event_view["timestamp_hour"] = col.dt.tz_offset(col_1).hour
-grouped = event_view.groupby(
+view = event_view.copy()
```

Comment:
Event view graph node is used by 
* project nodes (to project `event_timestamp` and `tz_offset`)
* assign node

Since it is used by other nodes, a copy is created before updating the frame.